### ISSUE TITLE
Change Req.: Please couple Sound to Threshold-values

### ISSUE THREAD

**Issue by rranke** [Tucsky/SignificantTrades#35]
Actually, until now i did not figure out when exactly the "special" "BTCUSD 1M BUY/SELL"-Sound is played by the app when using other pairs like "TRXM19".

Normally, i would expect, the sounds are **coupled** to the **threshold** values:

- First white dot: **quiet** sound
- Second white dot: **louder** sound
- Third white dot: **special 1M** sound
- Fourth white dot: **super special** sound

But it seems, that the "special 1M" sound (that is played in "BTCUSD") is generally only played at the amount of 1M TRX/XRP/whatever. Is it hardcoded for 1M in general?

Example:
If i would have a configuration like this, I never hear the "special 1M" sound that i would like to hear at 500K :-( 
So it should be played at the threshold-value of the third-dot (at the amount of "500K"). 
![Bildschirmfoto vom 2019-04-30 17:51:22](https://user-images.githubusercontent.com/5783257/56975222-9dd4f800-6b70-11e9-81c6-d979ff60f442.png)

**Comment by Tucsky** [comment]
Good point, actually I never enable the audio on other pairs than btcusd, and my thresholds are default so I always hear 2 notes at 100k 4 bips chord at 1M. I didn't expected it to be hardcoded until now. 

From what I can see in the code, the 4 bips sound is played when a trade 10x the 2nd threshold occur (1M = 100k*10)
https://github.com/Tucsky/SignificantTrades/blob/master/src/services/sfx.js#L48
_factor_ is `size / (2nd threshold * exchange multipler)`

place your 100k threshold at 50k and you should hear it at 500... lol

Can't remember why I did that tbh and you are right hooking the sounds to the thresholds would make more sense 💯

**Comment by rranke** [comment]
> factor is size / (2nd threshold * exchange multipler)

Ufff...., i think the normal user would not expect this kind of logic :-(

--> Now i think, it makes even more sense to **combine threshold with sound** !!! 
This would be great :-)
